packages/ui/package.json

```diff
@@ -29,13 +29,13 @@
     "./*": {
       "import": {
         "types": "./dist/components/*.d.mts",
-        "default": "./dist/components/*.mjs",
-        "browser": "./dist/components/*.mjs"
+        "browser": "./dist/components/*.mjs",
+        "default": "./dist/components/*.mjs"
       },
       "require": {
         "types": "./dist/components/*.d.ts",
-        "default": "./dist/components/*.js",
-        "browser": "./dist/components/*.js"
+        "browser": "./dist/components/*.js",
+        "default": "./dist/components/*.js"
```

Comment:
This order mattered for rendering the components within theme-builder app.

Reply:
According to node docs, the `default` conditional export should always come last, so this is the correct order: https://nodejs.org/api/packages.html#conditional-exports